src/pages/Homepage.js

```diff
@@ -0,0 +1,29 @@
+import { nanoid } from '@reduxjs/toolkit';
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import Card from '../components/Card';
+
+const Homepage = () => {
+  const { elements } = useSelector((state) => state.elements);
```

Comment:
> * [ ]  I kindly suggest that you add a drop down menu or search bar in the home page to filter the items on the homepage because that is according to the project requirement.

Hello, 
I am using the cards to filter the items as per the design, there's no need to use a searchbar or drop down menu because the card themselves filter everytime you click on one of them.

Reply:
The filter is for the cards. The filter is for the homepage. So you can have a search bar to search through your cards. I know your cards are just five and don't require a filter but this is a project requirement. If you want to ask for a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

Reply:
- As requested by the previous review, kindly add the filter functionality.
Since you have five cards and each card has its name. implement a search filter so it returns the letter or name matched in the search field.

see the requirement here.

*filter*
![filter](https://user-images.githubusercontent.com/59397159/181865446-1a227e2d-f866-4d53-bff6-48d444bbf4fe.png)